### PR TITLE
Display Elevate widget in BGE

### PR DESCRIPTION
**Pull request by psadie-sfdo**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/lwc/geBatchGiftEntryTable/geBatchGiftEntryTable.html [hunk 1]
```diff
@@ -27,58 +27,56 @@ <h1 class="slds-text-title_bold">{title}</h1>
...
+                                         enable-infinite-loading
+                                         onloadmore={loadMoreData}
+                                         data-qa-locator="datatable Batch Gifts">
                     </lightning-datatable>
                 </div>
-            </div>
-            <div if:false={hasData}>
+            </template>
+            <template if:false={hasData}>
                 <c-util-illustration title="Looks like you haven't entered any gifts yet!"
```
Comment (voduyemi) [inline src/lwc/geBatchGiftEntryTable/geBatchGiftEntryTable.html:74]: I know this isn't part of your WI? Do you know why we're not using a custom label here?

File: src/lwc/geFormService/geFormService.js [hunk 2]
```diff
@@ -42,6 +43,24 @@ class GeFormService {
     fieldTargetMappings;
     donationFieldTemplateLabel;
 
+    getPaymentTransactionStatusEnums = async () => {
```
Comment (voduyemi) [inline src/lwc/geFormService/geFormService.js:46]: Love the move here to `geFormService`. I'm just a little worried about the `handleError` function here, In `geFormTokenizeCard`, the `handleError` is tailored toward payment/transaction errors and is a banner inside of the widget, but if we start using this we start popping up toast error messages instead.

Also, side note, should we be looking to move this handleError function to a more generic class? We keep importing `utilTemplateBuilder` everywhere.

**Review by voduyemi — COMMENTED** [review]
Looks good to me, just had a few comments around custom labels and reshuffling of methods.

File: src/lwc/geFormService/geFormService.js [hunk 3]
```diff
@@ -42,6 +43,24 @@ class GeFormService {
...
+        try {
+            return Object.freeze(
+                JSON.parse(await getPaymentTransactionStatusValues())
```
Comment (dfuller114) [inline src/lwc/geFormService/geFormService.js:49]: Adding this comment from our discussion this morning for posterity. What do you think about creating an ElevateTransactionStatus javascript class that freezes an object containing hard coded values? I know Javascript and enums don't quite work the same way, but I think my motivation for doing so would moreso be to maintain consistency in matching types between client and server. I'm thinking ideally if we see a type in Apex we know is being used in the UI then we can know there's a corresponding javascript class for that type.

**Review by dfuller114 — COMMENTED** [review]
Everything looks good to me. Added the comment about ElevateTransactionStatus from our conversation earlier to finalize on that. This isn't a blocker though.

File: src/lwc/geFormService/geFormService.js (hunk 3 above)
Comment (dfuller114) [reply src/lwc/geFormService/geFormService.js:49]: @kenlewis-sfdo curious to get your thoughts on this.

Comment (psadie-sfdo) [reply src/lwc/geFormService/geFormService.js:49]: @dfuller114 @kenlewis-sfdo  - I merged this into the parent branch. Please drop comments into that branch for the full PR review. Thank you both!

**Pull request merged**